### DISCUSSION TITLE
Difference between optimizeMap() and make.acmap()?

### DISCUSSION THREAD

**Discussion by drserajames** [acorg/Racmacs#164] [Q&A] [answered]
Hi Sarah. Thanks for kind help. Could please let me know that what's the difference between optimizeMap() and make.acmap()?

_Originally posted by @judyssister in https://github.com/acorg/Racmacs/issues/130#issuecomment-1687721242_

**Comment by drserajames** [comment, marked as answer]
Hi @judyssister - I also made this a new issue as it's a new topic.

From the help file for make.acmap

`This is a wrapper function for first making a map with table data then, running optimizations to make the map otherwise done with acmap() followed by optimizeMap()`

Basically, make.acmap takes your titre data and makes in into the right format for running optimizeMap. If you have your titre data already saved in the right format, you can just read it in and use optimizeMap. Or if you have used make.acmap and you want to run some more optimisations on your map, you can use optimizeMap.

Hope this helps. Please reopen this issue if this isn't clear.

**Comment by judyssister** [comment]
Thanks, Sarah. Your response always helps me a lot :)

**Comment by drserajames** [comment]
Great! Please keep reaching out when you have issues. We really want people to use Racmacs to do good science.
